src/app/shared/services/pool/staking-pool-facade.ts

```diff
@@ -21,11 +21,21 @@ export class StakingPoolFacade {
     return from(this.pool.putStake(stake));
   }
 
+  getStartDate() {
+    return from(this.pool.getStart());
+  }
+
+  getEndDate() {
+    return from(this.pool.getEnd());
+  }
+
   checkReward() {
+    console.log('check reward');
```

Comment:
Is this intentional or missed here? Better to remove console.logs before merging but maybe you need to check something after it is deployed?

Reply:
missed, thanks for catching this!